react/components/ProductBrand/README.md

```diff
@@ -30,7 +30,13 @@ This component has an interface that describes which rules must be implemented b
 
 ### Configuration
 
-:construction: :construction: :construction: 
+| Prop name | Type | Description |
```

Comment:
Do any of these props have default values?
If so, I think it would be nice to include them here